extensions/binding/org.eclipse.smarthome.binding.lirc/src/main/java/org/eclipse/smarthome/binding/lirc/internal/LIRCHandlerFactory.java

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.binding.lirc.internal;
+
+import static org.eclipse.smarthome.binding.lirc.LIRCBindingConstants.THING_TYPE_REMOTE;
+
+import java.util.HashMap;
+import java.util.Hashtable;
+import java.util.Map;
+import java.util.Set;
+
+import org.eclipse.smarthome.binding.lirc.LIRCBindingConstants;
+import org.eclipse.smarthome.binding.lirc.handler.LIRCBridgeHandler;
+import org.eclipse.smarthome.binding.lirc.handler.LIRCRemoteHandler;
+import org.eclipse.smarthome.binding.lirc.internal.discovery.LIRCRemoteDiscoveryService;
+import org.eclipse.smarthome.config.discovery.DiscoveryService;
+import org.eclipse.smarthome.core.thing.Bridge;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+import org.eclipse.smarthome.core.thing.ThingUID;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandlerFactory;
+import org.eclipse.smarthome.core.thing.binding.ThingHandler;
+import org.osgi.framework.ServiceRegistration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.collect.Sets;
+
+/**
+ * The {@link LIRCHandlerFactory} is responsible for creating things and thing
+ * handlers.
+ *
+ * @author Andrew Nagle - Initial contribution
+ */
+public class LIRCHandlerFactory extends BaseThingHandlerFactory {
+
+    @SuppressWarnings("unused")
+    private Logger logger = LoggerFactory.getLogger(LIRCHandlerFactory.class);
+
+    private final static Set<ThingTypeUID> SUPPORTED_THING_TYPE_UIDS = Sets.union(
```

Comment:
drop it and use `LIRCBindingConstants.SUPPORTED_THING_TYPES` instead where needed